### ISSUE TITLE
key assignments

### ISSUE THREAD

**Issue by dml360** [finalburnneo/FBNeo#529]
I would like to know if it is possible in any way to assign a function to more than one key (I would like to place the movement in both the directional and analog keys)

**Comment by dinkc64** [comment]
Yes, map the directionals at the top, then go down below the System functions (pause, state, etc) and there should be a copy of the directionals from above, now you can map them here too.  also, click the checkbox if you'd like to have that button/direction be autofire.
